pipeline/4/Jenkinsfile-tier-0.groovy

```diff
@@ -60,7 +34,45 @@ node(nodeName) {
     }
 
     timeout(unit: "HOURS", time: 6) {
-        parallel testStages
+	    stage('RPM Sanity suite RHEL-8') {
```

Comment:
As the file is going to have `rhel-8` as suffix or prefix, we can drop the suffix from the stage name. We could rename the stage name to `RPM sanity`

Reply:
Since we cannot rename the file as mentioned in the previous comment, should i still change this or leave it as is?